### PR TITLE
[nnfwapi] Update comments : nnfw_output_tensorinfo

### PR DESCRIPTION
**Pull request by wateret**
Add the behavior after calling `nnfw_run` and refine description.

ONE-DCO-1.0-Signed-off-by: Hanjoung Lee <hanjoung.lee@samsung.com>

### CODE REVIEW

File: runtime/onert/api/include/nnfw.h [hunk 1]
```diff
@@ -363,11 +363,14 @@ NNFW_STATUS nnfw_input_tensorinfo(nnfw_session *session, uint32_t index,
 /**
  * @brief     Get i-th output tensor info
  *
- * <p>Before {@link nnfw_prepare} is invoked, this function return tensor info in model,
- * so updated tensor info by {@link nnfw_apply_tensorinfo} is not returned.</p>
+ * <p>After {@link nnfw_load_model_from_file} and before {@link nnfw_prepare} is invoked, it returns
+ * tensor info in the model. The info called by {@link nnfw_set_input_tensorinfo} </p>
```
Comment (ragmani) [inline runtime/onert/api/include/nnfw.h:367]: Is `nnfw_set_input_tensorinfo` right?

Comment (wateret) [reply runtime/onert/api/include/nnfw.h:367]: Yes, it will replaces `nnfw_apply_tensorinfo` so I changed it.

Comment (hyunsik-yoon) [reply runtime/onert/api/include/nnfw.h:367]: `The info called by {@link nnfw_set_input_tensorinfo} </p>` seems redundant?

Comment (ragmani) [reply runtime/onert/api/include/nnfw.h:367]: I see.

**Pull request merged**